source/operations/concepts/erasure-coding.rst

```diff
@@ -13,7 +13,7 @@ Erasure Coding
 MinIO Erasure Coding is a data redundancy and availability feature that allows MinIO deployments to automatically reconstruct objects on-the-fly despite the loss of multiple drives or nodes in the cluster. 
 Erasure Coding provides object-level healing with significantly less overhead than adjacent technologies such as RAID or replication. 
 
-MinIO splits each new object into data and parity blocks, where parity blocks support reconstruction of missing or corrupted data blocks. 
+MinIO shards each new object into data and parity blocks, where parity blocks support reconstruction of missing or corrupted data blocks. 
```

Comment:
Still don't like `blocks` here, but it may be a lost battle with it occurring in some many places. 😄

Reply:
doing the partition shard bit here too